runtime/gc_include/ObjectAccessBarrierAPI.hpp

```diff
@@ -351,16 +351,9 @@ class MM_ObjectAccessBarrierAPI
 	}
 
 	VMINLINE UDATA
-	mixedObjectGetDataSize(J9Class *objectClass, bool dontIncludeLockword)
+	mixedObjectGetDataSize(J9Class *objectClass)
 	{
-		UDATA size = objectClass->totalInstanceSize;
-		if (dontIncludeLockword) {
-			if ((0 <= (IDATA)objectClass->lockOffset)) {
-				size -= sizeof(j9objectmonitor_t);
-			}
-		}
-
-		return size;
+		return objectClass->totalInstanceSize;
```

Comment:
This is MM_MixedObjectModel::getSizeInBytesWithoutHeader

Reply:
So before this was only messing with the size for value type objects since that was the only place where `dontIncludeLockword` was true?

Reply:
Before, VTs were all getting lockwords. Now they are never getting them, so this is the same as the default.

Reply:
I dug a bit deeper and I am ok with this.